corehq/apps/app_manager/tests/test_submodule_xform.py

```diff
@@ -0,0 +1,112 @@
+from mock import patch
+from django.test import SimpleTestCase
+from corehq.apps.app_manager.const import APP_V2
+from corehq.apps.app_manager.models import (
+    AdvancedModule,
+    Application,
+    FormActionCondition,
+    LoadUpdateAction,
+    Module,
+    OpenSubCaseAction,
+    UpdateCaseAction,
+)
+from corehq.apps.app_manager.tests import TestFileMixin
+
+
+class SubmoduleXFormBase(TestFileMixin):
+    file_path = ('data', 'submodule_xform')
+    child_module_class = None
+
+    def setUp(self):
```

Comment:
These tests might be less code if you use `AppFactory`: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/tests/app_factory.py#L6-6

(would be good to get your feedback on it too)